tests/drum/test_units.py

```diff
@@ -96,6 +108,70 @@ def test_unsupervised(self):
 projectID = "abc123"
 
 
+@pytest.fixture
+def basic_model_metadata_yaml():
+    return {
+        "name": "basic-model-metadata-yaml",
+        "type": "inference",
+        "targetType": "regression",
+    }
+
+
+@pytest.fixture
+def complete_int_hyper_param():
+    return {
+        "name": "param_int",
+        "type": "int",
+        "min": 0,
+        "max": 1,
+        "default": 0,
+    }
+
+
+@pytest.fixture
+def complete_float_hyper_param():
+    return {
+        "name": "param_float",
+        "type": "float",
+        "min": 0.0,
+        "max": 1.0,
+        "default": 0.5,
+    }
+
+
+@pytest.fixture
+def complete_string_hyper_param():
+    return {
+        "name": "param_string",
+        "type": "string",
+        "default": "default string",
+    }
+
+
+@pytest.fixture
+def complete_select_hyper_param():
+    return {
+        "name": "param_select",
+        "type": "select",
+        "values": ["value 1", "value 2"],
+        "default": "value 1",
+    }
+
+
+@pytest.fixture
+def complete_multi_hyper_param():
+    return {
+        "name": "param_multi",
+        "type": "multi",
+        "values": {
+            "int": {"min": 0, "max": 1},
+            "float": {"min": 0.0, "max": 1.0},
+            "select": {"values": ["value 1", "value 2"]},
+        },
+        "default": "value 1",
+    }
```

Comment:
Please add a partial multi hyper param fixture with only int + select (or float + select)

Reply:
A testing was added to test this:
https://github.com/datarobot/datarobot-user-models/blob/e7ba94491c54a5091532ae3a8529eb46f06b4cb5/tests/drum/test_units.py#L769-L775